### ISSUE TITLE
tailwindcss support?

### ISSUE THREAD

**Issue by nothendev** [myFavShrimp/turf#16]
is there any way to use, for example, tailwind's `@apply` or `theme()` in turf modules?

**Comment by myFavShrimp** [comment]
No, this is currently not possible but I am not against the idea to provide support for tailwind.

If you want to use tailwindcss and turf together, you first need to process the tailwind styles (e.g. by calling the tailwind cli from a [`build.rs`](https://doc.rust-lang.org/cargo/reference/build-scripts.html) build script) and then refer to the resulting css files in turf's macros.
